src/main/java/org/folio/services/s3storage/FolioS3ClientFactory.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.services.s3storage;
+
+import org.folio.s3.client.FolioS3Client;
+import org.folio.s3.client.S3ClientFactory;
+import org.folio.s3.client.S3ClientProperties;
+
+public class FolioS3ClientFactory {
+
+  public static final String AWS_URL_CONFIG = "AWS_URL";
+  public static final String AWS_REGION_CONFIG = "AWS_REGION";
+  public static final String AWS_BUCKET_CONFIG = "AWS_BUCKET";
+  public static final String AWS_ACCESS_KEY_ID_CONFIG = "AWS_ACCESS_KEY_ID";
+  public static final String AWS_SECRET_ACCESS_KEY_CONFIG = "AWS_SECRET_ACCESS_KEY";
+  public static final String AWS_SDK_CONFIG = "AWS_SDK";
```

Comment:
I suggest to use these var names: https://github.com/folio-org/mod-bulk-operations?tab=readme-ov-file#environment-variables
I believe that all new implementations with S3 use this naming.

Reply:
renamed variables